templates/opt/stackstorm/chatops/st2chatops.env.erb

```diff
@@ -16,11 +16,17 @@ export HUBOT_LOG_LEVEL=<%= @hubot_log_level %>
 
 # Bot name
 export HUBOT_NAME=<%= @hubot_name %>
-export HUBOT_ALIAS='!'
+export HUBOT_ALIAS=<%= @hubot_alias %>
 
 ######################################################################
 # StackStorm settings
 
+# These parameters are default in this file, but can be overridden by
+# a global environment variable file.
+# To override these see /etc/sysconfig/st2chatops (RHEL/CentOS) or
+# /etc/default/st2chatops (Debian/Ubuntu). All of these files are
+# managed by Puppet.
+
```

Comment:
I think this template needs further changes, so
`export ST2_WEBUI_URL=https://${ST2_HOSTNAME}`
becomes
`export ST2_WEBUI_URL="${ST2_WEBUI_URL:-https://${ST2_HOSTNAME}}"`

and similar so every time env vars defined in `/etc/default/st2chatops` and `/etc/sysconfig/st2chatops` will have a priority over `st2chatops.env`.